### PR TITLE
Big fix 05 2021

### PR DESCRIPTION
**Pull request by azurras**
Uhhh.... I made a lot of changes. App still works tho

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,21 +4,21 @@ A simple password manager created in 2013.
...
+    mvn clean package
 ```
 
 ## Running
 
 1. Run the following to open jar
 
 ```bash
-    java -jar target/buddypasswordmanager-1.0-SNAPSHOT-jar-with-dependencies.jar
+    java -jar target/buddypasswordmanager-{version_number}-SNAPSHOT-jar-with-dependencies.jar
```
Comment (gavination) [inline README.md:23]: good.

File: src/main/java/com/appbuddy/buddypasswordmanager/gui/HelpGUI.java [hunk 2]
```diff
@@ -13,14 +13,15 @@
 import javax.swing.JTextArea;
 
 /**
- * @author Gavin Bauman - Christopher Bell - Gregory Degruy - Adrian
- *         Smith-Thompson updated: 5/28/2013
+ * @author Gavin Bauman
```
Comment (gavination) [inline src/main/java/com/appbuddy/buddypasswordmanager/gui/HelpGUI.java:16]: Good.

File: src/main/java/com/appbuddy/buddypasswordmanager/gui/BPMMenuBar.java [hunk 3]
```diff
@@ -0,0 +1,112 @@
+package com.appbuddy.buddypasswordmanager.gui;
```
Comment (gavination) [inline src/main/java/com/appbuddy/buddypasswordmanager/gui/BPMMenuBar.java:1]: Oh my, a real UI!

File: src/main/java/com/appbuddy/buddypasswordmanager/utils/BPMUtils.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+ */
+public class BPMUtils {
+    public static void populateTable() {
+        int row = 0;
+        int col = 0;
+        String nameOfUser = System.getProperty("user.name");
+        System.out.println("The name of the user is " + nameOfUser);
+        String entry;
+
+        File filePath = new File("/Users/" + nameOfUser + "/Documents/Buddy Information.txt");
```
Comment (gavination) [inline src/main/java/com/appbuddy/buddypasswordmanager/utils/BPMUtils.java:20]: string interpolation would be nice here.

**Review by gavination — APPROVED** [review]
LGTM. Much better than what was there previously. Added some comments for flavor and one minor suggestion. Give any thought to unit testing the methods with branching code paths? Me either lol.

File: src/main/java/com/appbuddy/buddypasswordmanager/utils/BPMUtils.java (hunk 4 above)
Comment (azurras) [reply src/main/java/com/appbuddy/buddypasswordmanager/utils/BPMUtils.java:20]: It would be... maybe next App Buddy day...lol

**Pull request merged**